README.md

```diff
@@ -1,27 +1,23 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+![]()
 ​
 
-# Portfolio: Setup and movile first
+# Portfolio: Fire Music Camp
 
-> iin this project I:
-
-- I set up a new repository and prepare it for development using best practices (e.g. linters).
-- Build the first 2 sections of the mobile website.
+> In this project,  I created a web page for a Music Camp.
   ​
   ​
 
 ## Built With
 
-​
-
-- Major languages: Html&css
+- Major languages: Html, scss, javascript
+- Libraries: bootstrap
 - Frameworks: Visual studio
 - Technologies used
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://carreraprogrammer.github.io/carrera_portfolio/)
-​
+- [Live Demo Link](https://carreraprogrammer.github.io/carrera_portfolio/)
```

Comment:
- [ ] Nice work done on your project, kindly fix the link to the Live version of your project is going to your portfolio page instead.